src/fiatExchanges/SelectProvider.tsx

```diff
@@ -82,6 +83,20 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
     }
   }, [fiatConnectQuotesError])
 
+  const asyncExchanges = useAsync(async () => {
```

Comment:
We probably want to wait for `asyncExchanges` to finish loading before removing the spinner and displaying the page; see line 132 for how we do this with `asyncProviders` currently.